index.html

```diff
@@ -57,6 +57,266 @@ <h1 id="primary-text">
+                            <p>CSS</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>HTML</p>
+                        </div>
+                    </div>
+                    <div class="bottom">
+                        <div class="see-btn">
+                            <p>See Project</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="work-card">
+                <div class="top-area">
+        
+                </div>
+                <div class="bottom-area">
+                    <div class="top">
+                        <p class="text">
+                            Multi-post stories<br>
+                            Gain+Glory
+                        </p>
+                    </div>
+                    <div class="tech">
+                        <div class="tech-card">
+                            <p>Ruby on Rails</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>CSS</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>HTML</p>
+                        </div>
+                    </div>
+                    <div class="bottom">
+                        <div class="see-btn">
+                            <p>See Project</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="work-card">
+                <div class="top-area">
+        
+                </div>
+                <div class="bottom-area">
+                    <div class="top">
+                        <p class="text">
+                            Multi-post stories<br>
+                            Gain+Glory
+                        </p>
+                    </div>
+                    <div class="tech">
+                        <div class="tech-card">
+                            <p>Ruby on Rails</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>CSS</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="tech-card">
+                            <p>HTML</p>
+                        </div>
+                    </div>
+                    <div class="bottom">
+                        <div class="see-btn">
+                            <p>See Project</p>
+                        </div>
+                    </div>
+                </div>
```

Comment:
- [ ] Overall, kindly examine this comparison please make corrections to match this design.  

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/243127471-931abb0a-db40-4b6c-8611-659c503fcbe3.png)|![image](https://user-images.githubusercontent.com/99715640/243127480-9ba9bd43-a0ed-4760-a34e-3487acf58244.png)|

  - [ ] It seems though you have double padding value on all project cards which is why your project is smaller in width, and making think things to stuffed up and slightly messy. Kindly check and reduce from `48px` to `24px`.